scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
Yes that is what I am suggesting., @MichaelArestad's breakpoints were at amounts higher than what we eventually agreed on breakpoints, and 400 seems a bit small to me for a breakpoint for the smallest font size.